general/getting_started.rst

```diff
@@ -1,2 +1,30 @@
 Getting started
 ===============
+
+Overview
+-----------------------
+
+OPM is a set of tools working above Nagios (or similar)
+to store into a database the perfdata retrieved by agents
+and display them in a web dashboard.
+
+check_pgactivity is a Nagios agent dedicated to PostgreSQL.
+It is technically separated from OPM.
+
+
+
+Requirements & Installation
+-----------------------------
+
+We suppose that you have in place and configured to your need:
+* a working Nagios (or similar ) installation
+* check_pgactivity (<https://github.com/OPMDG/check_pgactivity>)
```

Comment:
on a local build of the doc,  the `<` and `>` remains.  I'm not specialist of rst syntax, but the syntax doesn't look good, and the list is wrong.